collectors/diskspace.plugin/metadata.yaml

```diff
@@ -88,8 +88,24 @@ modules:
         examples:
           folding:
             enabled: true
-            title: ""
-          list: []
+            title: Config
+          list: |
+            - name: Basic
+              description: Default configuration.
```

Comment:
I think this is a very confusing example - at least give a proper name/description to it, it should reflect and briefly describe the set of options you suggested to use in `config`. If you can't give it a proper name/description - don't add it.

Reply:
I didn't know what and where it was requested (there is no link), just commented from a user perspective. Didn't get how that example is of any use, but ok.

Reply:
@sashwathn what is the idea? How do I use this example? What is "Basic" about oit?  What is that set of options? Or the options are irrelevant and the idea is just to show how the config looks like?

Reply:
@ilyam8 it was a requirement using Slack, I updated the description and I modified the name. 
The goal was to bring what we have in our README for this file.

Reply:
@ilyam8 : @Ancairon and I were running an exercise to identify what information from our existing readmes / learn is missing in the newly generated integrations and this configuration happens to be one of them.
We could call it “Configuration Options Example”?